docs/reference/setup/important-settings/discovery-settings.asciidoc

```diff
@@ -14,19 +14,20 @@ each other and elect a master node.
 
 Out of the box, without any network configuration, Elasticsearch will bind to
 the available loopback addresses and will scan local ports 9300 to 9305 to try
-to connect to other nodes running on the same server. This provides an auto-
-clustering experience without having to do any configuration.
+to connect to other nodes running on the same server. This provides an
+auto-clustering experience without having to do any configuration.
 
-When you want to form a cluster with nodes on other hosts, you must use the
+When you want to form a cluster with nodes on other hosts, you should use the
 `discovery.seed_hosts` setting to provide a list of other nodes in the cluster
 that are master-eligible and likely to be live and contactable in order to seed
-the <<modules-discovery-hosts-providers,discovery process>>. This setting
-should normally contain the addresses of all the master-eligible nodes in the
-cluster.  This setting contains either an array of hosts or a comma-delimited
-string. Each value should be in the form of `host:port` or `host` (where `port`
-defaults to the setting `transport.profiles.default.port` falling back to
-`transport.port` if not set). Note that IPv6 hosts must be bracketed. The
-default for this setting is `127.0.0.1, [::1]`.
+the <<modules-discovery-hosts-providers,discovery process>>. This setting should
+be a list of the addresses of all the master-eligible nodes in the cluster, each
+of which can either be an IP address or a hostname that {es} resolves to IP
+addresses using DNS. Note that IPv6 addresses must be bracketed.
```

Comment:
Suggestion:

> This setting should contain an address for each master-eligible node in the
cluster. Each address can be either an IP address or a hostname which resolves
to an IP string.

Reply:
Yes, two sentences are nicer here, thanks.